.editorconfig

```diff
@@ -0,0 +1,3 @@
+root = true
+[*.cs]
+end_of_line = lf
```

Comment:
I've added this file because I was having trouble with inconsistent line ending. 

I can remove this file if necessary. But I think having an editorconfig file greatly improves the dev experience. This one is by no means complete.

Reply:
I'd prefer that this is added in a separate PR if you don't mind.

Reply:
That sounds reasonable. How about I remove this on the last commit, so I don't have to worry about accidentally committing inconsistent line endings?